runbotics/runbotics-orchestrator-ui/src/src-app/providers/Socket.provider.tsx

```diff
@@ -29,12 +42,49 @@ const SocketProvider: FC<SocketProviderProps> = ({ children, uri, shouldAttach }
         });
     }, [isAuthenticated, uri, shouldAttach]);
 
-    useEffect(() => () => {
-        socket?.disconnect();
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-    }, [isAuthenticated, shouldAttach]);
+    useEffect(
+        () => () => {
+            socket?.disconnect();
+        },
+        // eslint-disable-next-line react-hooks/exhaustive-deps
+        [isAuthenticated, shouldAttach]
+    );
+
+    useEffect(() => {
+        closeSnackbar();
+
+        setInterval(() => {
+            if (socket === null || !shouldAttach || !socket.disconnected) {
+                closeSnackbar('warning');
+                return;
+            }
+            if (socket.disconnected) {
+                enqueueSnackbar(
+                    translate('Scheduler.Dialog.NoServerToConnection'),
+                    {
+                        variant: 'warning',
+                        key: 'warning',
+                        preventDuplicate: true,
+                        persist: true,
+                        onExited: () =>
+                            enqueueSnackbar(translate('Scheduler.Dialog.ConnectionRestored'), {
+                                variant: 'success',
+                                preventDuplicate: true,
+                                autoHideDuration: 5000,
+                                key: 'restored',
+                            }),
+                    }
+                );
+            }
+        }, 1500);
```

Comment:
![image](https://github.com/runbotics/runbotics/assets/44121335/22bcf9a4-ace5-44aa-87de-a37ea6cb8718)

clear interval if its not needed anymore, if this will be called many times (and it will) it will stack intervals in the app